### PR TITLE
resizable div for ace_js

### PR DESCRIPTION
**Pull request by Joeper214**

### CODE REVIEW

File: app/assets/javascripts/cf_templates.js [hunk 1]
```diff
@@ -121,6 +121,16 @@
     }
   });
 
+  function resizeAce() {
+    
+    return $('#description').height($(window).height());
+  };
```
Comment (skyarch-networks) [inline app/assets/javascripts/cf_templates.js]: __[jshint]__
*  Unnecessary semicolon.

File: app/assets/stylesheets/bootstrap_extend.css [hunk 2]
```diff
@@ -159,6 +159,14 @@ input[type="number"] {
     box-shadow: none;
 }
 
+#resizable{
```
Comment (pocke) [inline app/assets/stylesheets/bootstrap_extend.css]: Why do you use `id`?
Can we use `class` instead of `id`?

Comment (Joeper214) [inline app/assets/stylesheets/bootstrap_extend.css]: :+1:

**Comment by Joeper214** [comment]
@pocke Please review and Merge PR. Thanks!     :bow:

**Comment by pocke** [comment]
OK :bow:

**Comment by pocke** [comment]
Code doesn't have any problem.
I'll merge it after operation check.

**Comment by pocke** [comment]
editor is under label. 
default
nohlsearch


I tweaked this at other branch. what do you think?
https://github.com/skyarch-networks/skyhopper/tree/resizable-div-tweak

**Comment by pocke** [comment]
tweaked
default

**Comment by Joeper214** [comment]
Thanks.!

**Comment by pocke** [comment]
I'll merge tweak branch into this PR, and merge PR!

**Pull request merged**